GcmEndpoints/README.md

```diff
@@ -7,7 +7,9 @@ This backend template employs [Google Cloud Endpoints](https://developers.google
 
 # 1. Adding a backend in Android Studio
 
-To add the backend to your existing Android app from this backend template, open Android Studio ([installation instructions](https://developer.android.com/sdk/installing/studio.html)) and navigate to "File &rarr; New Module..." or right-click on your project and choose "New &rarr; Module".
+Start with an existing Android app. You can use the [GCM sample Quickstart app](https://github.com/googlesamples/google-services/tree/master/android/gcm) as a starting point.
+
```

Comment:
Yea good point. My wording was a bit strong. How about "This guide builds on top of GCM sample Quickstart app. If you already have an existing application with GCM set up, you can start at section 2.3"?

I need to give it a one over to make sure the flow makes sense.